habitat_baselines/il/vln/dagger.py

```diff
@@ -343,12 +370,13 @@ def _update_dataset(self, data_it):
                             np.array([step[1] for step in ep], dtype=np.int64),
                             np.array([step[2] for step in ep], dtype=np.int64),
                         ]
-                        txn.put(
-                            str(start_id + collected_eps).encode(),
-                            msgpack_numpy.packb(
-                                transposed_ep, use_bin_type=True
-                            ),
-                        )
+                        with lmdb_env.begin(write=True) as txn:
```

Comment:
The `txn` was moved here as that'll cause it to write each thing immediately and reduces RAM usage (I was scared of ooming on 60GB).